code/renderers/react/src/entry-preview.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import semver from 'semver';
+import type { Decorator } from './public-types';
+
+export const parameters = { renderer: 'react' };
+export { render } from './render';
+export { renderToCanvas } from './renderToCanvas';
+export { mount } from './mount';
+
+export const decorators: Decorator[] = [
+  (Story, context) => {
+    if (!context.parameters?.react?.rsc) return <Story />;
```

Comment:
Can you change it to `experimentalRsc`? and update any documentation related to this, also potentially deprecate the main.js flag?

Reply:
Do we still want to call it `experimental`? @shilman